pegasus/sites.v3/code.org/public/css/tools.css

```diff
@@ -78,12 +78,16 @@ h3.tutorial-info-h {
 .tutorial-info-start {
   margin-top: 20px;
 }
-
+.tutorial-info-start[dir='rtl']{
+  float:right !important;
```

Comment:
We try to avoid using `!important` as much as possible. Was this necessary?

Reply:
Actually this can be removed.

Reply:
I removed them.